rhcos4/profiles/stig.profile

```diff
@@ -0,0 +1,23 @@
+documentation_complete: true
```

Comment:
if this is a draft, why not set `documentation_complete: false`?

Reply:
@JAORMX Need it to build so that updated transmission to DISA can happen.